pkg/prometheus/metrics.go

```diff
@@ -59,11 +72,24 @@ func init() {
 	legacyregistry.MustRegister(authFormCounterResult)
 	legacyregistry.MustRegister(authBasicCounter)
 	legacyregistry.MustRegister(authBasicCounterResult)
+	legacyregistry.MustRegister(X509MissingSANCounter)
 
 	for _, resultLabel := range []string{SuccessResult, FailResult, ErrorResult} {
 		authBasicCounterResult.WithLabelValues(resultLabel)
 		authFormCounterResult.WithLabelValues(resultLabel)
 	}
+
+	for _, provider := range [...]configv1.IdentityProviderType{
```

Comment:
I really don't want to open pandora's box... but that list would fit nicely into the api? :laughing:

Reply:
i am fine tackling this is in an initiative tracking >=4.10